.changeset/shiny-ravens-march.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/color-picker": patch
+---
+
+add aria-valuetext
```

Comment:
Change "add aria-valuetext" to ”added `aria-valuetext` to `AlphaSlider` and `HueSlider`.".

Reply:
Fixed.